indico/modules/receipts/default_templates/receipt.yaml

```diff
@@ -0,0 +1,3 @@
+default_filename: receipt
+title: Default Receipt
```

Comment:
Is the "Default" in the title actually useful/needed? In the end people will copy it into an event/category, and then having this in the title no longer makes sense...